packages/samples/angular/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "dev": "npm run copy:assets && ng serve",
+    "dev": "npm run build:assets && dev:angular",
```

Comment:
The angular sample doesn't have a `build` script, therefore it doesn't run when the CI does `npm run build`. Should I add it?